Shared/Store/CredentialProviderStore.swift

```diff
@@ -81,11 +84,22 @@ extension CredentialProviderStore {
     }
 
     private func refresh() {
+        Observable.combineLatest(self.dataStore.list, self.accountStore.oauthInfo, self.accountStore.profile)
```

Comment:
nit: this seems worthy a comment to note it's signaling there's already a list to display while waiting for the refresh ...

Reply:
alternately: for clarity here, I would probably do something like:
```
Observable.combineLatest(self.accountStore.oauthInfo.filterNotNull(), self.accountStore.profile.filterNotNull())
       .flatMap { self.dataStore.list }
       .filter { $0.isEmpty }
       .bind ......
```

but either way works. I did have to read the many `$0.1` type annotations a few times to understand what's going on.